Makefile

```diff
@@ -131,8 +131,19 @@ docker_push: docker_tag docker_auth ## push to docker hub
 	docker push 'openedx/blockstore:latest-newrelic'
 	docker push "openedx/blockstore:${GITHUB_SHA}-newrelic"
 
+COMMON_CONSTRAINTS_TXT=requirements/common_constraints.txt
+.PHONY: $(COMMON_CONSTRAINTS_TXT)
+$(COMMON_CONSTRAINTS_TXT):
+	wget -O "$(@)" https://raw.githubusercontent.com/edx/edx-lint/master/edx_lint/files/common_constraints.txt || touch "$(@)"
+
 upgrade: export CUSTOM_COMPILE_COMMAND=make upgrade
-upgrade: ## update the requirements/*.txt files with the latest packages satisfying requirements/*.in
+upgrade: $(COMMON_CONSTRAINTS_TXT)	## update the requirements/*.txt files with the latest packages satisfying requirements/*.in
+	sed 's/pyjwt\[crypto\]<2.0.0//g' requirements/common_constraints.txt > requirements/common_constraints.tmp
```

Comment:
We are creating a local copy of the common constraints file and override/remove the global constraint which we don't need. This approach is used temporarily and will be improved in [BOM-2721](https://openedx.atlassian.net/browse/BOM-2721)